feature-rich/package.json

```diff
@@ -7,57 +7,57 @@
   "license": "GPL-2.0-or-later",
   "private": true,
   "devDependencies": {
-    "@ckeditor/ckeditor5-adapter-ckfinder": ">=36.0.0",
-    "@ckeditor/ckeditor5-alignment": ">=36.0.0",
-    "@ckeditor/ckeditor5-autoformat": ">=36.0.0",
-    "@ckeditor/ckeditor5-basic-styles": ">=36.0.0",
-    "@ckeditor/ckeditor5-block-quote": ">=36.0.0",
-    "@ckeditor/ckeditor5-ckbox": ">=36.0.0",
-    "@ckeditor/ckeditor5-ckfinder": ">=36.0.0",
-    "@ckeditor/ckeditor5-clipboard": ">=36.0.0",
-    "@ckeditor/ckeditor5-cloud-services": ">=36.0.0",
-    "@ckeditor/ckeditor5-code-block": ">=36.0.0",
-    "@ckeditor/ckeditor5-core": ">=36.0.0",
-    "@ckeditor/ckeditor5-dev-translations": ">=33.0.0",
-    "@ckeditor/ckeditor5-dev-utils": ">=33.0.0",
-    "@ckeditor/ckeditor5-editor-classic": ">=36.0.0",
-    "@ckeditor/ckeditor5-engine": ">=36.0.0",
-    "@ckeditor/ckeditor5-enter": ">=36.0.0",
-    "@ckeditor/ckeditor5-essentials": ">=36.0.0",
-    "@ckeditor/ckeditor5-export-pdf": ">=36.0.0",
-    "@ckeditor/ckeditor5-export-word": ">=36.0.0",
-    "@ckeditor/ckeditor5-find-and-replace": ">=36.0.0",
-    "@ckeditor/ckeditor5-font": ">=36.0.0",
-    "@ckeditor/ckeditor5-heading": ">=36.0.0",
-    "@ckeditor/ckeditor5-highlight": ">=36.0.0",
-    "@ckeditor/ckeditor5-horizontal-line": ">=36.0.0",
-    "@ckeditor/ckeditor5-html-embed": ">=36.0.0",
-    "@ckeditor/ckeditor5-html-support": ">=36.0.0",
-    "@ckeditor/ckeditor5-image": ">=36.0.0",
-    "@ckeditor/ckeditor5-import-word": ">=36.0.0",
-    "@ckeditor/ckeditor5-indent": ">=36.0.0",
-    "@ckeditor/ckeditor5-language": ">=36.0.0",
-    "@ckeditor/ckeditor5-link": ">=36.0.0",
-    "@ckeditor/ckeditor5-list": ">=36.0.0",
-    "@ckeditor/ckeditor5-media-embed": ">=36.0.0",
-    "@ckeditor/ckeditor5-mention": ">=36.0.0",
-    "@ckeditor/ckeditor5-page-break": ">=36.0.0",
-    "@ckeditor/ckeditor5-paragraph": ">=36.0.0",
-    "@ckeditor/ckeditor5-paste-from-office": ">=36.0.0",
-    "@ckeditor/ckeditor5-remove-format": ">=36.0.0",
-    "@ckeditor/ckeditor5-select-all": ">=36.0.0",
-    "@ckeditor/ckeditor5-special-characters": ">=36.0.0",
-    "@ckeditor/ckeditor5-style": ">=36.0.0",
-    "@ckeditor/ckeditor5-table": ">=36.0.0",
-    "@ckeditor/ckeditor5-theme-lark": ">=36.0.0",
-    "@ckeditor/ckeditor5-typing": ">=36.0.0",
-    "@ckeditor/ckeditor5-ui": ">=36.0.0",
-    "@ckeditor/ckeditor5-undo": ">=36.0.0",
-    "@ckeditor/ckeditor5-upload": ">=36.0.0",
-    "@ckeditor/ckeditor5-utils": ">=36.0.0",
-    "@ckeditor/ckeditor5-widget": ">=36.0.0",
-    "@ckeditor/ckeditor5-word-count": ">=36.0.0",
-    "@webspellchecker/wproofreader-ckeditor5": "^2.0.1",
+    "@ckeditor/ckeditor5-adapter-ckfinder": "^37.1.0",
```

Comment:
Any reason for changing the version qualifier from `>=` to `^`? Given we have `resolutions` section (in some) it doesn't really matter.

Reply:
`>=` updates to any version minor or major and that rids us of control if there are any changes we need to adress for the demos to work properly. This is in keeping the public demos consistent with private repo: ckeditor/ckeditor5-private-demos#127

Reply:
ok, let's keep it consistent between the projects :+1: